tests/autoyast/installation.pm

```diff
@@ -224,7 +228,7 @@ sub run {
                 send_key_until_needlematch 'create-partition-plans-finished', $cmd{ok};
                 next;
             }
-            @expected_warnings = grep { !match_has_tag($_) } @expected_warnings;
+            @processed_warnings = grep { match_has_tag($_) } @expected_warnings;
```

Comment:
What happens here if the @expected_warnings is not defined? Based on https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/10515/files#diff-61c3977e3aad390a1a1af99f7b6aa75bR138 the variable is not always presented.

Reply:
grep will return an empty array. You can try following code snippet:
```perl
my @expected_warnings;
my @processed_warnings = grep { defined($_) } @expected_warnings;

print(scalar @processed_warnings);
```

This will give you `0`